localization/languages/nl.json

```diff
@@ -0,0 +1,251 @@
+{
+    "name": "Dutch)",
+    "identifier": "nl-BE",
```

Comment:
How similar is Belgian Dutch to Dutch in the Netherlands? Should this set of translations be used in the Netherlands as well?